client/schema/v0/index.js

```diff
@@ -20,6 +20,11 @@ export default {
             name: 'title',
             type: 'text'
           },
+          {
+            name: 'holder',
+            label: 'Licence holder',
+            type: 'holder'
+          },
```

Comment:
Does this cause any side-effects when editing a legacy project?

Reply:
@lennym - ugh, It does ...